openverse_api/env.docker

```diff
@@ -16,3 +16,5 @@ UPSTREAM_DATABASE_PORT="5432"
 SEMANTIC_VERSION="1.0.0"
 
 ELASTICSEARCH_URL="es"
+
+SENTRY_DSN="https://08f4706d16004f57bcd37eb907bfc2e7@o787041.ingest.sentry.io/6107216"
```

Comment:
Looks redundant to have this env var here and in the `env.example` too, in fact, I had to remove the Django variables from here in order to turn off the DEBUG mode. Can we leave this file only for docker-specific variables?

Reply:
Whoops! I misunderstood the reason for the `env.docker`. I'll remove it.